src/Components/Projects/Form/index.js

```diff
@@ -8,181 +9,141 @@ import FunctionalButton from '../../Shared/Buttons/FunctionalButton';
 import { useSelector, useDispatch } from 'react-redux';
 import { createProject, editProject } from '../../../redux/projects/thunks';
 import { setFetching } from '../../../redux/projects/actions';
+import { Select } from 'Components/Shared';
+import { getEmployees } from 'redux/employees/thunks';
 
 const ProjectForm = () => {
   const urlValues = window.location.search;
   const urlParams = new URLSearchParams(urlValues);
   const id = urlParams.get('id');
   const idRegEx = /^(?=[a-f\d]{24}$)(\d+[a-f]|[a-f]+\d)/i;
   const isValidId = idRegEx.test(id);
-  const { children, modalTitle, fetching } = useSelector((state) => state.projects);
   const dispatch = useDispatch();
-  const [projectBody, setProjectBody] = useState({
+  const { children, modalTitle, fetching } = useSelector((state) => state.projects);
+  const { list: employeesList } = useSelector((state) => state.employees);
+  const roles = ['QA', 'DEV', 'TL'];
+  const [modalDisplay, setModalDisplay] = useState('');
+  const [values, setValues] = useState({
     name: '',
     description: '',
     clientName: '',
     startDate: '',
     endDate: '',
+    employees: [
+      {
+        role: '',
+        employee: '',
+        rate: ''
+      }
+    ],
     active: true
   });
-  const [employees, setEmployees] = useState([]);
-  const [modalDisplay, setModalDisplay] = useState('');
+
+  const { register, handleSubmit, reset, control } = useForm({
+    mode: 'onChange',
+    defaultValues: values
+  });
+
+  const { fields, append, remove } = useFieldArray({
+    control,
+    name: 'employees'
+  });
 
   useEffect(async () => {
+    dispatch(getEmployees());
     if (isValidId) {
       dispatch(setFetching(true));
       try {
         const response = await fetch(`${process.env.REACT_APP_API_URL}/projects/${id}`);
         const data = await response.json();
-        console.log('project data', data);
-        setProjectBody({
+        console.log(data);
```

Comment:
```suggestion
        console.log(data);
```
This log should be deleted!